jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -373,7 +373,11 @@
             # Get golang into our PATH so we can run e2e.go
             export PATH=${{PATH}}:/usr/local/go/bin
 
-            timeout -k 15m 55m ./hack/jenkins/e2e-runner.sh && rc=$? || rc=$?
+            if [[ -e "./hack/jenkins/.use_dockerized_e2e" ]]; then
+              timeout -k 15m 55m ./hack/jenkins/dockerized-e2e-runner.sh && rc=$? || rc=$?
```

Comment:
there's really no place we can define a function that would be sourced by all of the places we use this.

I think long-term we want to put the timeout into the docker image or e2e.go anyway.